spacecmd/src/spacecmd/utils.py

```diff
@@ -100,14 +100,21 @@ def parse_command_arguments(command_args, argument_parser, glob=True):
         return leftovers, opts
     except IndexError:
         return None, None
+ 
+def is_interactive(options):
+    """
+    check if any named options were passed to the function, and if so,
+    declare that the function is non-interactive
+    note: because we do it this way, default options are not passed into
+    OptionParser, as it would make determining if any options were passed
+    too complex
 
+        Parameters:
+            options (dict): A dictionary
 
-# check if any named options were passed to the function, and if so,
-# declare that the function is non-interactive
-# note: because we do it this way, default options are not passed into
-# OptionParser, as it would make determining if any options were passed
-# too complex
-def is_interactive(options):
+        Returns:
+            bool: Returns false if a key in options has value else returns true otherwise
```

Comment:
You could add a bit of semantics for the description of returned value (in addition to its type), for example "returns false if named options were passed".